### PR TITLE
feat: Dockerfile added

### PR DESCRIPTION
**Pull request by ratik**
so here are some features
1. Dockerfile
2. docker-compose
3. new make points
4. updated docs

Make sure you checked out cosmos-query-relayer to https://github.com/neutron-org/cosmos-query-relayer/pull/19

### CODE REVIEW

**Review by pr0n00gler — DISMISSED** [review]
Tested, LGTM.

But could you update README pls with instructions how to run this stuff using docker and maybe create additional make command like: `make start-docker`?

**Review by pr0n00gler — DISMISSED** [review]

**Comment by zavgorodnii** [comment]
ehhh the tests don't pass, right?

**Comment by ratik** [comment]
> ehhh the tests don't pass, right?

fixed

File: readme.md [hunk 1]
```diff
@@ -43,6 +43,31 @@ make start-rly
...
+make build-docker-image
+```
+
+Then you can start/stop with
+```shell
+make start-docker-container
+make stop-docker-container
+```
+
+## Running with docker + relayer
```
Comment (zavgorodnii) [inline readme.md:59]: wow!

**Review by zavgorodnii — APPROVED** [review]

**Review by pr0n00gler — APPROVED** [review]

**Pull request merged**